Source/ACE/Network/Handlers/AuthenticationHandler.cs

```diff
@@ -29,9 +29,18 @@ public static class AuthenticationHandler
         public static void HandleLoginRequest(ClientPacket packet, Session session)
         {
             PacketInboundLoginRequest loginRequest = new PacketInboundLoginRequest(packet);
-            log.Info($"First 2 of token: {Convert.ToByte(loginRequest.JwtToken[0])}, {Convert.ToByte(loginRequest.JwtToken[1])})");
-            Task t = new Task(() => DoLogin(session, loginRequest));
-            t.Start();
+            if (!string.IsNullOrEmpty(loginRequest.JwtToken))
```

Comment:
1) as of .net 4, the proper check is string.IsNullOrWhitespace
2) should probably check for len >= 2 as well
3) I'm fine with removing this debug statement.  it's holdover from when i implemented things.